tensorly/cp_tensor.py

```diff
@@ -300,6 +300,50 @@ def cp_normalize(cp_tensor):
     return CPTensor((weights, normalized_factors))
 
 
+def cp_fix_sign(cp_tensor, mode=0, func=None):
```

Comment:
Maybe `cp_flip_sign` to echo the SVD version? Just a thought, happy to keep the current name.